modules/remotecontrol/RemoteControl.js

```diff
@@ -45,6 +74,9 @@ class RemoteControl {
         this._initialized = true;
```

Comment:
Well at the time I wasn't sure why some singleton instance variables were being set in init and some in the constructor.  I just wanted to make sure the choice of placement was intentional.